service/project.go

```diff
@@ -0,0 +1,95 @@
+package service
+
+import (
+	"context"
+
+	"release-manager/pkg/apierrors"
+	"release-manager/pkg/dberrors"
+	"release-manager/service/model"
+
+	"github.com/google/uuid"
+)
+
+type ProjectService struct {
+	authSvc    model.AuthService
+	repository model.ProjectRepository
+}
+
+func NewProjectService(authSvc model.AuthService, repo model.ProjectRepository) *ProjectService {
+	return &ProjectService{
+		authSvc:    authSvc,
+		repository: repo,
+	}
+}
+
+func (s *ProjectService) Create(ctx context.Context, pc model.ProjectCreation, authUserID uuid.UUID) (model.Project, error) {
+	if err := s.authSvc.AuthorizeManagerRole(ctx, authUserID); err != nil {
+		return model.Project{}, err
+	}
+
+	p, err := model.NewProject(pc.Name, pc.SlackChannelID, pc.RlsAlertCfg)
+	if err != nil {
+		return model.Project{}, apierrors.NewProjectUnprocessableError().Wrap(err).WithMessage(err.Error())
+	}
+
+	return s.repository.Create(ctx, p)
+}
+
+func (s *ProjectService) Get(ctx context.Context, projectID uuid.UUID, authUserID uuid.UUID) (model.Project, error) {
+	// TODO add project member authorization
```

Comment:
Authorization will be added in the next PR, where project members feature will be added.